ocaml/gencert/selfcert.ml

```diff
@@ -131,3 +137,16 @@ let xapi_pool ~uuid pemfile =
   selfsign issuer extensions key_length expire_days pemfile
   >>| (fun _ -> ())
   |> R.failwith_error_msg
+
+let xapi_cluster ~cn =
+  (* in practical terms, never expire *)
+  let expire_days = 365 * 2000 in
+  let key_length = 2048 in
```

Comment:
Can we share these settings with the one used for xapi itself?

Reply:
We need to implement certificate rotation for the cluster to do that safely.